forui/lib/src/widgets/tabs/tabs.dart

```diff
@@ -0,0 +1,168 @@
+  }
+
+  @override
+  State<FTabs> createState() => _FTabsState();
+}
+
+class _FTabsState extends State<FTabs> with SingleTickerProviderStateMixin {
+  late int _selectedTab;
+  late final FTabController _controller;
+
+  @override
+  void initState() {
+    super.initState();
+    _selectedTab = widget.initialIndex;
+    _controller = FTabController(length: widget.tabs.length, vsync: this);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = context.theme;
+    final typography = theme.typography;
+    final style = widget.style ?? context.theme.tabsStyle;
+    final tabs = widget.tabs;
+    final materialLocalizations =
+        Localizations.of<MaterialLocalizations>(context, MaterialLocalizations);
+
+    // ignore_for_file: no_leading_underscores_for_local_identifiers
+    final _tabs = Material(
+      color: Colors.transparent,
+      child: Column(
+        children: [
+          DecoratedBox(
+            decoration: style.decoration,
+            child: TabBar(
+              tabs: [
+                for (final tab in tabs)
+                  Tab(
+                    height: style.height,
+                    child: tab.rawLabel ?? Text(tab.label!),
+                  )
+              ],
+              controller:
+                  widget.controller?._controller ?? _controller._controller,
+              isScrollable: widget.scrollable,
+              padding: style.padding,
+              indicator: style.indicator,
+              indicatorSize: style.indicatorSize,
+              unselectedLabelStyle: style.unselectedLabel.scale(typography),
+              dividerColor: Colors.transparent,
+              labelStyle: style.selectedLabel.scale(typography),
+              onTap: (index) {
+                setState(() {
+                  _selectedTab = index;
+                });
+                widget.onTap?.call(_selectedTab);
+              },
+            ),
+          ),
+          SizedBox(height: style.spacing),
+          // A workaround to ensure any widgets under Tabs do not revert to material text style
+          // TODO: abstract out logic
+          DefaultTextStyle(
+            style: theme.typography.toTextStyle(
+              fontSize: theme.typography.base,
+              color: theme.colorScheme.foreground,
+            ),
+            child: tabs[_selectedTab].content,
+          ),
+        ],
+      ),
+    );
+
+    return materialLocalizations == null
+        ? Localizations(
+            locale: Localizations.maybeLocaleOf(context) ??
+                const Locale('en', 'US'),
+            delegates: const [
+              GlobalMaterialLocalizations.delegate,
+              GlobalWidgetsLocalizations.delegate,
+              GlobalCupertinoLocalizations.delegate,
+            ],
+            child: _tabs,
+          )
+        : _tabs;
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+    _controller.dispose();
```

Comment:
```suggestion
    _controller.dispose();
    super.dispose();
```

For future reference, super.dispose() should always be the last since disposing it first results in errors esp with animation-related stuff.